docs/handbook/roles.mdx

```diff
@@ -2,5 +2,46 @@
 title: 👾 Character Classes & Roles
 ---
 
+#### 🤗 How to play Receptor:
 
-#### *TODO*
+- Welcome new members, ask them about themselves and refer them to the #signpost
+- Check the battle plan to see who's working on what, and ask them how is that going.
+
+#### 🎨 How to play Artist:
+
+- Get into the Memebox - https://www.dropbox.com/sh/wm523tie1mgqg01/AAAGbobhz6a1fIqsrC-vd8ZEa?dl=0
+  - Take .psd of any art you see there & upgrade any part of it
+  - Make any new art
+
+#### 🔎 How to play Bughunter:
+
+- Fix (if possible) and report all broken links
+- Report misalignments in articles or other content
+- Report what's incoherent or hard to understand
+
+#### 🤠 How to play Headhunter:
+
+- Refer people to MetaGame
+
+#### 📜 How to play Shiller:
+
+- read here https://forum.metagame.wtf/t/shilling-school/26
+
+#### 👾 How to play The Propagator
+
+- Take posts from here: https://tinyurl.com/v8w9a9b (there might be new ones, check here https://metagame.substack.com/ and add there
+- Distribute these blog posts to 3rd party sites, eg. Reddit, steemit, hackernoon or wherever might be suitable.
+- Fill in the distribution sheet with the link, and report to #🌱-claim-seeds
+
+#### 🖊️ How to play synthesizer:
+
+- Read the Gitcoin grant description https://gitcoin.co/grants/213/metagame
+- Read the old Path to 1.0 Doc - https://tinyurl.com/rdpxkqv
```

Comment:
the links here seems very old, maybe we should not mention any links on this page as they are getting out of sync very fast, or there should be single point of truth page with all links that we point to from different places